src/SDKs/Azure.ApplicationModel.Configuration/data-plane/README.md

```diff
@@ -1,13 +1,50 @@
 # Overview 
 
 Azure.ApplicationModel.Configuration is a component of the .NET Azure SDK. 
-It provides APIs for storing and retrieving application settings.
+It provides APIs for Microsoft [Azure's App Configuration Service](https://docs.microsoft.com/en-us/azure/azure-app-configuration/) which is s a managed configuration service that allows to easily store and manage all application settings in one central place that is separated from code.
+
+Developers can use this SDK to interact with the [Configuration Store](https://docs.microsoft.com/en-us/azure/azure-app-configuration/quickstart-dotnet-core-app#create-an-app-configuration-store) where the configuration settings are stored.
+
+Actions that can be executed are:
+
+- Perform basic reads, writes, updates, and deletes of an application configuration settings.
+- Get the history of a configuration setting.
+- Watch for changes in a specific configuration setting.
 
 # Installing
 
 A NuGet package called Azure.ApplicationModel.Configuration will be avaliable soon.
 
+# Configuration Setting
+Is the fundamental resource within a Configuration Store. In its simplest form it is a key and a value. However, there are additional properties such as the modifiable content type and tags fields that allow the value to be interpreted or associated in different ways.
+
+The Label property of a configuration setting provides a way to separate configuration settings into different dimensions. These dimensions are user defined and can take any form. Some common examples of dimensions to use for a label include regions, semantic versions, or environments. Many applications have a required set of configuration keys that have varying values as the application exists across different dimensions.
+For example, MaxRequests may be 100 in "NorthAmerica", and 200 in "WestEurope". By creating a configuration setting named MaxRequests with a label of "NorthAmerica" and another, only with a different value, in the "WestEurope" label, a solution can be achieved that allows the application to seamlessly retrieve configuration settings as it runs in these two dimensions.
+
+Properties of a Configuration Setting:
+
+```c#
+    string Key { get; set; }
+
+    string Label { get; set; }
+
+    string Value { get; set; }
+
+    string ContentType { get; set; }
+
+    string ETag { get; }
+
+    DateTimeOffset LastModified { get; }
+
+    bool Locked { get; }
+
+    IDictionary<string, string> Tags { get; set; }
+```
+
 # Hello World
+The following example demonstrates how to initialize a ConfigurationClient and perform a basic operations in the Configuration Store.
+
+To begin using ConfigurationClient, a connection string must be provided which specifies the configuration store endpoint and credentials to use when sending requests. From that point, the example uses the client to set, retrieve and delete a configuration setting by its name.
```

Comment:
Should we add a note saying where the connection string can be aquired? i.e. in the protal

Reply:
We could. the Hello world examples says that it  can get it from the portal, but we are not mentioning the Azure CLI.